http-client-core/src/main/java/io/micronaut/http/client/HttpClientConfiguration.java

```diff
@@ -314,6 +318,8 @@ public void setExceptionOnErrorStatus(boolean exceptionOnErrorStatus) {
     }
 
     /**
+     * [not in java.net HttpClient].
```

Comment:
why is this only for netty?

Reply:
Fixed in https://github.com/micronaut-projects/micronaut-core/pull/8441/commits/1d85bc88b501c2af65fe06616de24ff8f2e85a12 and https://github.com/micronaut-projects/micronaut-core/pull/8777/commits/596807a53f579eaf423bec7da1b7c84875708723